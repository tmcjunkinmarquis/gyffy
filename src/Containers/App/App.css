.App {
  height: 100vh;
  width: 100vw;
  background-color: #E8EEF1;
  text-align: left;
}


