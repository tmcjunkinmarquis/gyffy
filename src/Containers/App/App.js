import React, { Component } from 'react';
import SearchForm from '../SearchForm/SearchForm';
import './App.css';
import FuelStationList from '../FuelStationList/FuelStationList';
import FilterMenu from '../../Components/FilterMenu/FilterMenu';

// <FilterModalMenu />
class App extends Component {

  render() {
    return (
      <div className="App">
        <header className="App-header">
          
          <h1 className="gyffy">gyffy</h1>
          <SearchForm />
          <FilterMenu />
          <FuelStationList />
        </header>
      </div>
    );
  }
}

export default App;
