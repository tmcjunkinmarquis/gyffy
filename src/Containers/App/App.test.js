import React from 'react';
// import ReactDOM from 'react-dom';
import App from './App';
import { shallow } from 'enzyme';

describe('', ()=>{

  it('should match the snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});


