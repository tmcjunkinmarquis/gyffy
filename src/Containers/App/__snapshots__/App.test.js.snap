// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <header
        className="App-header"
>
        <h1
                className="gyffy"
        >
                gyffy
        </h1>
        <Connect(SearchForm) />
</header>,
      "className": "App",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="gyffy"
>
            gyffy
</h1>,
          <Connect(SearchForm) />,
        ],
        "className": "App-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "gyffy",
            "className": "gyffy",
          },
          "ref": null,
          "rendered": "gyffy",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "header",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <header
          className="App-header"
>
          <h1
                    className="gyffy"
          >
                    gyffy
          </h1>
          <Connect(SearchForm) />
</header>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="gyffy"
>
              gyffy
</h1>,
            <Connect(SearchForm) />,
          ],
          "className": "App-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "gyffy",
              "className": "gyffy",
            },
            "ref": null,
            "rendered": "gyffy",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "header",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
